README.md

```diff
@@ -78,44 +77,31 @@ Example command:
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
-### Install
+clone this repo with this link https://github.com/TayyabKM/Vet_clinic.git
 
-Install this project with:
-
-<!--
-Example command:
+then use the following command:
 
 ```sh
-  cd my-project
-  gem install
+ git clone https://github.com/TayyabKM/Vet_clinic.git
 ```
---->
+
+### Install this project with:
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  cd Vet_clinic
+  code .
 ```
```

Comment:
- [ ] [optional] @TayyabKM great work implementing the changes requested, however it would be nice adding commands in the user like
 Use [schema.sql](https://github.com/microverseinc/curriculum-template-databases/blob/main/schema.sql) to create all tables.
Use [data.sql](https://github.com/microverseinc/curriculum-template-databases/blob/main/data.sql) to populate tables with sample data.
Check [queries.sql](https://github.com/microverseinc/curriculum-template-databases/blob/main/queries.sql) for examples of queries that can be run on a newly created database